rqt_topic crashes

how to reproduce:
1. rosrun rqt_topic rqt_topic
2. select a topic

this gives an error:
[ERROR] [1487596624.452584, 40.374000]: bad callback: <bound method TopicInfo.message_callback of <rqt_topic.topic_info.TopicInfo object at 0x7f94480baed0>>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 720, in _invoke_callback
    cb(msg)
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rqt_topic/topic_info.py", line 100, in message_callback
    self.sizes.append(buff.len)
AttributeError: 'cStringIO.StringO' object has no attribute 'len'

--- a/rqt_topic/src/rqt_topic/topic_info.py
+++ b/rqt_topic/src/rqt_topic/topic_info.py
@@ -97,7 +97,7 @@ class TopicInfo(ROSTopicHz):
             # time consuming workaround...
             buff = StringIO()
             message.serialize(buff)
-            self.sizes.append(buff.len)
+            self.sizes.append(len(buff.getvalue()))
 
             if len(self.timestamps) > self.window_size - 1:
                 self.timestamps.pop(0)

